app/models/similar_occupation_standards.rb

```diff
@@ -29,37 +33,56 @@ def similar_to
   private
 
   def query
-    {
-      size: RESULTS_SIZE,
-      min_score: MINIMUM_SCORE,
-      query: {
-        bool: {
-          should: [
-            {match: {
-              title: {query: occupation_standard.title, boost: 5}
-            }},
-            {match: {
-              work_process_titles: {
-                query: occupation_standard.work_processes.pluck(:title).to_sentence
-              }
-            }},
-            {match: {
-              ojt_type: {query: occupation_standard.ojt_type, boost: 0.5}
-            }},
-            {match: {
-              state: {query: occupation_standard.registration_agency&.state&.abbreviation}
-            }}
-          ],
-          minimum_should_match: 1,
-          must_not: [
-            {
-              term: {
-                _id: occupation_standard.id
-              }
-            }
-          ]
+    definition = Search.new
+    should = []
+    should.append(
+      Queries::Match.new(
+        title: {
+          query: occupation_standard.title, boost: 5
+        }
+      )
+    )
+
+    should.append(
+      Queries::Match.new(
+        work_process_titles: {
+          query: occupation_standard.work_processes.pluck(:title).to_sentence
+        }
+      )
+    )
+
+    should.append(
+      Queries::Match.new(
+        ojt_type: {
+          query: occupation_standard.ojt_type,
+          boost: 0.5
+        }
+      )
+    )
+
+    if occupation_standard.registration_agency&.state
+      should.append(
+        Queries::Match.new(
+          state: {
+            query: occupation_standard.registration_agency&.state&.abbreviation
+          }
```

Comment:
could this be turned into a method for getting the state abbreviation? Then you could call that method and make this line easier to read.

Reply:
@crystal-williams-brown good point! I'll create a method in the model